app/services/npq/application/reject.rb

```diff
@@ -16,6 +16,9 @@ def call
         return self unless valid?
 
         npq_application.update!(lead_provider_approval_status: "rejected")
+        if FeatureFlag.active?(:npq_capping) && npq_application.funded_place?
```

Comment:
ah this is the wrong service, can we please only do this on the `ChangeToPending` service and set this to `nil` please